sdk/storage/azure-storage-queue/README.md

```diff
@@ -15,6 +15,39 @@ A single queue message can be up to 64 KB in size, and a queue can contain milli
 
 ### Include the package
 
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```

Comment:
> Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.